common/src/main/java/com/viaversion/viaversion/protocols/v1_21_9to1_21_11/rewriter/BlockItemPacketRewriter1_21_11.java

```diff
@@ -75,6 +78,16 @@ public void registerPackets() {
         });
     }
 
+    @Override
+    public Item handleItemToClient(final UserConnection connection, final Item item) {
```

Comment:
handleItemToClient is replaced with handleItemDataComponentsToClient in newer protocols (since items need to be hashed)